src/marketplace/ListingsView.js

```diff
@@ -76,27 +89,25 @@ class ListingsView extends React.Component<Props, State> {
 				obj['amount'] = jsonObjectOfSells[sale].amount;
```

Comment:
So why not make saleObject a class? Then you don't have to be messing with object[string] and you can just call the constructor and not worry about all of this?